notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
Changed the BUCKET_NAME to BUCKET_URI.